CONTRIBUTING.md

```diff
@@ -0,0 +1,162 @@
+# How to Modify the Specification
+
+Welcome!  This document is a meta-discussion of how the specifications
+should be safely altered when you want to include some amazing new
+functionality.
+
+Please remember that we're all trying to Make Things Better.  Respect,
+consideration, kindness and humor have made this process
+[fun](00-introduction.md#theme-song) and rewarding and we'd like to keep it
+that way.  We're nice!
+
+## Extension Design
+
+There are several extension mechanisms in the spec; you should seek to use
+them, or introduce new ones if necessary.
+
+### Adding New Inter-Peer Messages
+
+Unknown odd inter-peer messages are ignored, aka "it's OK to be odd!"
+which makes more sense as you get to know me.
+
+If your message is an enhancement, and you don't need to know if the other
+side supports it, you should give it an odd number.  If it would be broken
+if the other side doesn't support it (ie. Should Never Happen) give it an
+even number.  Mistakes happen, and future versions of the software may well
+not be tested against ancient versions.
+
+If you want to experiment with new message types internally, I recommend
```

Comment:
nit : maybe points to where message types are laid out like in [lightning message format](https://github.com/lightningnetwork/lightning-rfc/blob/master/01-messaging.md#lightning-message-format)